rhoas-client/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+### OIDC Clients for RHOAS
+quarkus.oidc-client.client-enabled=false
+# Red Hat SSO (refresh token read from "event-bridge.rhoas.sso.red-hat.refresh-token" property)
+quarkus.oidc-client.red-hat-sso.auth-server-url=${event-bridge.rhoas.sso.red-hat.auth-server-url}
+quarkus.oidc-client.red-hat-sso.client-id=cloud-services
```

Comment:
@kostola These should be injected as environment parameters and not hardcoded

Reply:
I set this as a fixed value because it's what the production Red Hat SSO auth flow expects, but it can be overridden either with the property or the env var.
Should I define a dedicated `event-bridge.rhoas.sso.red-hat.client-id` property for this? Is the property name OK to you?

Reply:
Variable name is fine for me. Given that this is sensitive information, it should be encrypted at rest in our Git repo. Take a look at https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/tree/main/kustomize#sealed-secrets on how to do that.

If you need any help, just give me a shout.

Reply:
I defined the variable and added the secrets :+1: